account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/SendOtpNotificationHandler.java

```diff
@@ -169,6 +175,12 @@ public APIGatewayProxyResponseEvent sendOtpRequestHandler(
             return generateApiGatewayProxyResponse(500, GENERIC_500_ERROR_MESSAGE);
         }
 
+        incrementUserSubmittedCredentialIfNotificationSetupJourney(
+                cloudwatchMetricsService,
+                JourneyType.ACCOUNT_MANAGEMENT,
+                sendNotificationRequest.getNotificationType().name(),
+                configurationService.getEnvironment());
```

Comment:
I think this could work here for now. I'm slightly hesitant just because we're not conditioning this if `sendNotificationRequest.getNotificationType()` is `VERIFY_EMAIL` or `VERIFY_PHONE_NUMBER`.

Should another notification type be supported by this lambda in the future, we may get some extra metrics recorded that don't make sense in context (i.e. they may not be about "user submitting a credential").

Reply:
Redacted, you are already doing this with `notificationSetupTypes.contains(notificationType)` in `EmailCheckMetricsHelper